package.json

```
@@ -26,7 +26,7 @@
     "graphql-js"
   ],
   "engines": {
-    "node": "^16.19.0 || ^18.14.0 || >=19.7.0"
+    "node": ">=20.7.0 || >=20.0.0 || >=22.0.0 || >=23.0.0 || >=24.0.0"
```

```suggestion
    "node": ">=20.19.0 || >=22.12.0 || >=23.0.0 || >=24.0.0"
```

These are the versions where require(ESM) support was added:

https://nodejs.org/en/blog/release/v20.19.0
https://nodejs.org/en/blog/release/v22.12.0